src/redis/stack.rs

```diff
@@ -0,0 +1,51 @@
+use testcontainers::{core::WaitFor, Image};
+
+const NAME: &str = "redis/redis-stack-server";
+const TAG: &str = "7.2.0-v8";
+
+#[derive(Debug)]
+pub struct RedisStack;
```

Comment:
Let's add documentation as for others, preferably for both `Redis` and `RedisStack`.
It was missing only because this module was moved from the old repository